devutils/src/devutils/invoke_utils.py

```diff
@@ -320,15 +320,6 @@ def check_tag_version(ctx):
             )
             sys.exit(1)
 
-    @task
-    def set_version_in_deps(ctx, version):
-        """Sets a new version of this project in the project dependencies"""
-        root_pyproject = root / "pyproject.toml"
-        assert root_pyproject.exists(), f"Expected {root_pyproject} to exist."
-
-        all_pyprojects = list(collect_deps_pyprojects(root_pyproject))
-        all_pyprojects.append(root_pyproject)
-
```

Comment:
Not needed anymore. (Slack [thread](https://robocorptechnologies.slack.com/archives/C05CQR7JLNT/p1687343211057589?thread_ts=1687302523.420139&cid=C05CQR7JLNT))